modules/seo/JsonLD.tsx

```diff
@@ -2,7 +2,7 @@ export const JsonLD: React.FC<{ jsonObject: unknown }> = ({ jsonObject }) => (
   <script
     type="application/ld+json"
     dangerouslySetInnerHTML={{
-      __html: JSON.stringify(jsonObject),
+      __html: JSON.stringify(jsonObject).replaceAll('</', '<\\/'),
```

Comment:
Maybe this remediation should be applied everywhere `dangerouslySetInnerHTML` is used ? Even if it's used with constant data it's always a good idea to remove bad patterns.

So applying this modification here:

* https://github.com/lyonjs/lyonjs.github.com/blob/nonce-and-checks/app/layout.tsx#L29
* https://github.com/lyonjs/lyonjs.github.com/blob/nonce-and-checks/modules/header/LyonJSHead.tsx#L63

Reply:
Yes, I thought we were using JsonLD component to but we wan't due to next head handling, on it